http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala

```diff
@@ -156,6 +156,7 @@ class LeoMetricsMonitor[F[_]](config: LeoMetricsMonitorConfig,
                 authHeader = Authorization(Credentials.Token(AuthScheme.Bearer, token))
                 relayPath = Uri.unsafeFromString(baseUri.asString) / app.appName.value
                 isUp <- serviceName match {
+                  // should this be removed too?
                   case ServiceName("wds") =>
```

Comment:
wasnt sure if this should be removed as well, since what what I can see WDS is still a member of the CoA class but it is just being set up differently. possible that I may be misunderstanding.

Reply:
I don't think it will have the same `relayPath` as `wds/` doesn't proceed the status check anymore (e.g. there is no proxy re-routing the request)

Nevertheless, might be worthwhile implementing a similar metrics-monitor for WDS -- perhaps out of scope for this ticket though, I think